web/frontend/src/components/slam-2d-render.vue

```diff
@@ -110,16 +193,47 @@ const updateCloud = (pointcloud: Uint8Array) => {
   intersectionPlane.position.set(center.x, 0, center.z);
   raycaster.objects = [intersectionPlane];
 
+  // construct grids
+  const axesHelper1 = new THREE.AxesHelper( 5 );
```

Comment:
These axes and grid helpers could get pretty distracting / introduce noise for someone not doing motion planning and instead wanting to just inspect a PCD. We should make them toggle-able with a prime [switch](https://prime.viam.com/?path=/docs/elements-switch--docs) before we ship this. LMK if you need help there!